site/content/en/docs/Guides/metrics.md

```diff
@@ -66,6 +66,14 @@ Follow the [Stackdriver Installation steps](#stackdriver-installation) to see yo
 | agones_k8s_client_workqueue_longest_running_processor | How long the longest running workqueue processor has been running in microseconds                                                                                                           | gauge     |
 | agones_k8s_client_workqueue_unfinished_work_seconds   | How long unfinished work has been sitting in the workqueue in seconds                                                                                                                       | gauge     |
 
+### Dropping Metric Labels
```

Comment:
This section should be behind a feature code since this alpha feature won't exist until the next release.